weave/integrations/anthropic/anthropic_sdk.py

```diff
@@ -103,12 +102,38 @@ async def _async_wrapper(
         return add_accumulator(
             op,  # type: ignore
             make_accumulator=lambda inputs: anthropic_accumulator,
-            should_accumulate=should_use_accumulator,
         )
 
     return wrapper
 
 
+def anthropic_stream_accumulator(
+    acc: typing.Optional["Message"],
+    value: "MessageStream",
+) -> "Message":
+    from anthropic.lib.streaming._types import MessageStopEvent
+    
+    print(f"    type(value): {type(value)}")
+    if acc is None:
+        acc = {}
+    if isinstance(value, MessageStopEvent):
+        print(f"Here we go!")
+        acc = value
```

Comment:
does this imply the entire payload is here?

Reply:
yeap =)